docs/management/saved-objects/saved-object-ids.asciidoc

```diff
@@ -0,0 +1,87 @@
+[[saved-object-ids]]
+=== Saved Object IDs
+
+In the past, many saved object types could have the same ID in different <<xpack-spaces,spaces>>. For example, if you copied dashboard "123"
+from the one space to another space, the second dashboard would also have an ID of "123". While the saved object ID is not something
+that users would interact with directly, many aspects of {kib} rely on it, notably URLs. If you have a "deep link" URL to a saved dashboard,
+that URL includes the saved object ID.
+
+**Starting in the 8.0 release**, {kib} requires most saved objects to have _globally unique_ IDs. This is a change that we needed to make to
+support sharing saved objects to multiple spaces. Most saved objects cannot be shared to multiple spaces _yet_, but we needed to start
+enforcing globally unique object IDs first.
+
+We have made several enhancements to minimize the impact to users, and this document describes what you need to know about the changes and
+how it will affect you.
```

Comment:
nit/simplify

```suggestion
We have made several enhancements to minimize the impact, and this document describes what you need to know about the changes and
how it will affect you.
```